index.html

```diff
@@ -13,5 +13,41 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Acme Web Sulutions</title>
   </head>
-  <body></body>
+  <body>
+    <header class="site-header">
+      <div class="site-wrapper">
+        <div>header</div>
+      </div>
+    </header>
+    <section class="first-section">
+      <div class="site-wrapper">
+        <div>1</div>
+      </div>
+    </section>
+    <section class="second-section">
+      <div class="site-wrapper">
+        <ul class="column-container">
+          <li class="column-item">1</li>
+          <li class="column-item">2</li>
+          <li class="column-item">3</li>
+        </ul>
+      </div>
+    </section>
+    <div class="site-wrapper">
+      <section class="third-section">1</section>
+    </div>
+    <div class="site-wrapper">
+      <ul class="fourth-section">
+        <li class="third-section-item">
+          <div>1</div>
+        </li>
+        <li class="third-section-item">
+          <div>2</div>
+        </li>
+      </ul>
+      <footer class="site-footer">
```

Comment:
The footer should be outside of the site wrapper because we need a black background for it.

Reply:
You mean we don't need wrapper here?
The wrapper is already inside the footer.

Reply:
Yes but before that the footer by itself is in the upper wrapper. Look at the line 39.

Reply:
I removed line 39.